modules/imgcodecs/src/grfmt_jpeg2000_openjpeg.cpp

```diff
@@ -0,0 +1,343 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Could you please use [short license header](https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure)?

```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
//
// Copyright (C) 2020, Stefan Brüns <stefan.bruens@rwth-aachen.de>
```

Reply:
done